BoostLibraryList/.env

```
@@ -0,0 +1 @@
+GITHUB_TOKEN=<github_token here...>
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)